doc/central_scheduler.rst

```diff
@@ -78,20 +78,20 @@ The task history has the following pages:
 
     .. figure:: history.png
        :alt: Recent history screenshot
-* ``/history/by_id/:id``
+* ``/history/by_id/id``
   detailed information about a run, including:
   parameter values, the host on which it ran, and timing information.
   Example screenshot:
 
     .. figure:: history_by_id.png
        :alt: By id screenshot
-* ``/history/by_name/:name``
-  a listing of all runs of a task with the given task name.
+* ``/history/by_name/name``
+  a listing of all runs of a task with the given task ``name``.
   Example screenshot:
 
     .. figure:: history_by_name.png
        :alt: By name screenshot
-* ``/history/by_params/:name?data=params``
```

Comment:
The colon refers to the `name` being a path parameter, we should revert it to having the `:`

Reply:
> we should revert it to having the :

Not sure what do you mean. The `:` is already in the current doc. Even if you reject my PR, no reversion is necessary.